src/diffusers/modeling_utils.py

```diff
@@ -21,7 +21,9 @@
 import torch
 from torch import Tensor, device
 
-from diffusers.utils import is_accelerate_available
+import accelerate
```

Comment:
This is fine because if `torch` is installed then accelerate has to be installed as well